app/models/support_interface/vendor_api_monitor.rb

```diff
@@ -0,0 +1,66 @@
+module SupportInterface
+  class VendorAPIMonitor
+    def never_connected
+      @_never_connected = providers.left_outer_joins(:vendor_api_requests)
+        .includes(:vendor_api_tokens)
+        .distinct
+        .where(vendor_api_requests: { id: nil })
+    end
+
+    def no_sync_in_24h
+      @_no_sync_in_24h ||= connected_providers.where.not(
+        id: VendorAPIRequest.syncs.select(:provider_id).distinct.where('created_at > ?', 24.hours.ago),
+      ).map do |provider|
+        ProviderWithAPIUsageStats.new(provider)
+      end
+    end
+
+    def no_decisions_in_7d
+      @_no_decisions_in_7d ||= connected_providers.where.not(
+        id: VendorAPIRequest.decisions.select(:provider_id).distinct.where('created_at > ?', 7.days.ago),
```

Comment:
same here re decisions, if a provider has sent a decision which wasn't valid should we include it?